frontend/src/components/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -1,9 +1,11 @@
 import { FC, UserSignUpRequestDto } from 'common/types/types';
-import { Button, Input } from 'components/common/common';
+import { Button, Input, Link } from 'components/common/common';
 import { getNameOf } from 'helpers/helpers';
 import { useAppForm } from 'hooks/hooks';
 import { userSignUp as userSignUpValidationSchema } from 'validation-schemas/validation-schemas';
 import { DEFAULT_SIGN_UP_PAYLOAD } from './common';
+import { AppRoute } from 'common/enums/enums';
+import styles from 'components/auth/components/styles.module.scss';
```

Comment:
```suggestion
import styles from './styles.module.scss';
```
you need to use own styles